scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Set the version information from Git and the current date.
+VERSION="1.0.0"
```

Comment:
```suggestion
VERSION="0.1.0"
```

Reply:
Semantic versioning : MAJOR.MINOR.PATCH
MAJOR: version changes indicate incompatible API changes,
MINOR: version changes add functionality in a backwards-compatible manner, and
PATCH: version changes are for backwards-compatible bug fixes.

The practice of starting a project's versioning at 0.1.0 rather than 1.0.0 reflects a development philosophy. Here's why projects often start at 0.1.0